packages/react/src/primitives/types/tabs.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseComponentProps, AriaProps } from './base';
+import { BaseStyleProps } from './style';
+import { FlexStyleProps } from './flex';
+import React from 'react';
+
+export type TabsGrow = 'equal' | 'relative';
+
+export interface TabsProps
+  extends BaseComponentProps,
+    AriaProps,
+    BaseStyleProps,
+    FlexStyleProps {
+  /**
+   * The Tabs component only accepts TabItem components as children.
+   */
+  children: React.ReactElement | React.ReactElement[];
+
+  /**
+   * Change which Tab content is initially displayed. Pass in the index of the Tab you wish to show. The default is index 0 (the first tab).
+   */
+  defaultTab?: number;
```

Comment:
Perhaps `defaultIndex` is a better name?

Reply:
+1 to `defaultIndex`